### PR TITLE
Added Ansible UTM Info Module for proxy auth profile entities.

### PR DESCRIPTION
**Pull request by maunzCache**
##### SUMMARY
This PR will add a new utm info module which allows the management of proxy auth profile entries of the Sophos UTM. This module integrates with the existing utm_utils .

##### ISSUE TYPE
* New Module Pull Request

##### COMPONENT NAME
* module/web_infrastructure/sophos_utm/utm_proxy_auth_profile_info.py

### CODE REVIEW

**Comment by getjack** [comment]
@stearz @RickS-C137 Please help out reviewing :)

**Review by MatrixCrawler — APPROVED** [review]

**Comment by MatrixCrawler** [comment]
shipit

File: lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_auth_profile_info.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        description:
+          - The name of the object. Will be used to identify the entry
+        required: true
+
+extends_documentation_fragment:
+    - utm
+"""
+
+EXAMPLES = """
+- name: Read UTM proxy_auth_profile
```
Comment (gundalow) [inline lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_auth_profile_info.py:42]: Does this need `register:` to be a real example?

Comment (gundalow) [reply lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_auth_profile_info.py:42]: @getjack Thoughts on this, is the example useful as-is?

Comment (gundalow) [reply lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_auth_profile_info.py:42]: @getjack @MatrixCrawler I'd welcome your thoughts on this.

**Comment by gundalow** [comment]
@getjack @MatrixCrawler Should this be a fact module?

**Pull request closed without merging**